src/main/java/edu/cornell/kfs/web/filter/CornellLoginFilter.java

```diff
@@ -0,0 +1,50 @@
+package edu.cornell.kfs.web.filter;
+
+import edu.cornell.cynergy.service.impl.CUWebAuthAuthenticationService;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletRequestWrapper;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class CornellLoginFilter implements Filter {
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
+        HttpServletRequest httpRequest = (HttpServletRequest)servletRequest;
+        HttpServletResponse httpResponse = (HttpServletResponse)servletResponse;
+
+        String remoteUser = getRemoteUser(httpRequest);
+        HttpServletRequestWrapper request =  new HttpServletRequestWrapper(httpRequest) {
+            @Override
+            public String getRemoteUser() {
+                return remoteUser;
+            }
+        };
+
+        filterChain.doFilter(request,httpResponse);
```

Comment:
Ideally, but since this is the only comment on this review, I'm willing to live with it if you are.